bitnami/rabbitmq/templates/statefulset.yaml

```
@@ -156,9 +156,8 @@ spec:
                   key: rabbitmq-password
             - name: RABBITMQ_PLUGINS
               value: {{ include "rabbitmq.plugins" . | quote }}
-            {{- range $key, $value := .Values.extraEnvVars }}
-            - name: {{ $key }}
-              value: {{ $value | quote }}
+            {{- if .Values.extraEnvVars }}
+            {{- tpl (toYaml .Values.extraEnvVars) . | nindent 12 }}
```

Thanks so much! Please use the `common.tplvalues.render` macro instead of `tpl (toYaml ...)`:
```suggestion
            {{- if .Values.extraEnvVars }}
            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
```